WordPress/Docs/Commenting/HooksInlineDocsStandard.xml

```diff
@@ -15,7 +15,7 @@
 do_action( 'hook' );
         ]]>
         </code>
-        <code title="Invalid: using a non-docblock comment">
+        <code title="Invalid: not using a DocBlock comment">
```

Comment:
A "DocBlock" is made up of two parts:

- the `/**  */` structure, known as a _DocComment_.
- the contents with Summary, optional description, tags, etc. known as the _PHPDoc_.

As such, the "DocBlock comment" here, which is referencing whether one asterisk or two is used, is really checking if it's a multi-line comment (`/*`) or a _DocComment_ (`/**`), hence my earlier suggestion of `Invalid: not using a DocComment."

(Note that the period is still missing at the end from your version.)